user_sessions/locale/ru/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,199 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-04-19 07:30+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: admin.py:20
+msgid "Is Valid"
+msgstr "Действующий"
+
+#: admin.py:25
+msgid "Active"
+msgstr "Активный"
+
+#: admin.py:26
+msgid "Expired"
+msgstr "Истекший"
+
+#: admin.py:37
+msgid "Owner"
+msgstr "Владелец"
+
+#: admin.py:42
+msgid "Self"
+msgstr "Сам"
+
+#: models.py:37
+msgid "session key"
+msgstr "ключ сессии"
+
+#: models.py:39
+msgid "session data"
+msgstr "сессия"
+
+#: models.py:40
+msgid "expiry date"
+msgstr "дата истечения срока"
+
+#: models.py:44
+msgid "session"
+msgstr "сессия"
+
+#: models.py:45
+msgid "sessions"
+msgstr "сессии"
+
+#: templates/user_sessions/session_list.html:5
+msgid "<em>unknown on unknown</em>"
+msgstr "<em>неизвестный из неизвестных</em>"
+
+#: templates/user_sessions/session_list.html:6
+msgid "<em>неизвестный</em>"
+msgstr ""
+
+#: templates/user_sessions/session_list.html:8
+msgid "Active Sessions"
+msgstr "Активные сессии"
+
+#: templates/user_sessions/session_list.html:13
+msgid "Location"
+msgstr "Место расположения"
+
+#: templates/user_sessions/session_list.html:14
+msgid "Device"
+msgstr "Устройство"
+
+#: templates/user_sessions/session_list.html:15
+msgid "Last Activity"
+msgstr "Последняя активность"
+
+#: templates/user_sessions/session_list.html:16
+#: templates/user_sessions/session_list.html:34
+#: templates/user_sessions/session_list.html:36
+msgid "End Session"
+msgstr "Конец сеанса"
+
+#: templates/user_sessions/session_list.html:25
+#, python-format
+msgid "%(time)s ago (this session)"
+msgstr "%(time)s назад (сессия)"
+
+#: templates/user_sessions/session_list.html:27
+#, python-format
+msgid "%(time)s ago"
+msgstr "%(time)s назад"
+
+#: templates/user_sessions/session_list.html:47
+msgid ""
+"You can also end all other sessions but the current.\n"
+"        This will log you out on all other devices."
+msgstr ""
+"Вы также можете завершить все другие сеансы, кроме текущего.\n"
+"        Вы выйдете из системы на всех других устройствах.."
+
+#: templates/user_sessions/session_list.html:49
+msgid "Завершить все остальные сеансы"
+msgstr ""
+
+#: templatetags/user_sessions.py:20
+msgid "Chrome"
+msgstr ""
```

Comment:
Excuse my ignorance: does an empty string fall back to the original or render the actual empty string?

Reply:
yes indeed, it falls back to the original tag!